pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.black]
+line-length = 120
+skip-string-normalization = true
+target-version = ['py27','py35','py36', 'py37', 'py38']
```

Comment:
EB still supports py26, doesn't it?

Reply:
Yes, but `black` doesn't. TBH I'm not sure that this is of any real use anyway.